CONTRIBUTING.md

```diff
@@ -25,10 +28,29 @@
 
 # Contributing to Athens
 
-Whether you are a designer, developer, or have other superpowers, please see our [v1 Project Board](https://github.com/athensresearch/athens/projects/2) to see what we're working on.
+## Developers
 
-- The best place to reach us is our [Discord](https://discord.gg/GCJaV3V)! 👾
+- Join our [Discord](https://discord.gg/GCJaV3V) 👾 and check out the `#engineering` and `#engineers` channels.
+  - Post work updates in the `#build-in-public`. This keeps the team on the same page! Let's avoid stepping on each others toes, minimize blockers / dependencies, and cheer each other on!
+- Watch the repo and bookmark our [Project Board](https://github.com/athensresearch/athens/projects/2). This is the ultimate source of truth for product roadmapping.
+- To start working on your PR, you have a few ways to get started:
+    1. ask a question in our `#engineering` [Discord](https://discord.gg/GCJaV3V) channel
+    1. comment on one of the existing top-level issues on the project board
+    1. create a PR draft or issue, then assign yourself (prefer drafts over new issues)
```

Comment:
Are we trying to get people to open empty draft PRs? I feel like when you're ready to open a (not empty) draft, you're almost 50% of the way there, and some time may have past already.

Reply:
I'm pretty okay with empty-ish PR drafts. That's the purpose of the line right below this. Some people might have a really clear intention of what they want to work on.

Also, creating a PR draft directly skips the need to open an issue as well. Perhaps slightly lighter weight!

Reply:
Fine by me then! :)